integration-tests/capacity/staking.test.ts

```diff
@@ -145,18 +143,58 @@ describe("Capacity Staking Tests", function () {
                         assert.equal(capacityStaked.totalTokensStaked, 1n * CENTS, "should return a capacityLedger with 1M total tokens staked");
                         assert.equal(capacityStaked.totalCapacityIssued, expectedCapacity, "should return a capacityLedger with 1/50M capacity issued");
                     });
+
+                    it("successfully increases the amount that was targeted to provider from different accounts", async function () {
+                        // Create a new account
+                        const additionalKeys = await createAndFundKeypair(accountBalance);
+
+                        // get the current account info
+                        let currentAcctInfo = await ExtrinsicHelper.getAccountInfo(additionalKeys.address);
+                        const currentStaked = (await firstValueFrom(ExtrinsicHelper.api.query.capacity.capacityLedger(stakeProviderId))).unwrap();
+
+                        await assert.doesNotReject(stakeToProvider(additionalKeys, stakeProviderId, 1n*CENTS));
+
+                        const capacityStaked = (await firstValueFrom(ExtrinsicHelper.api.query.capacity.capacityLedger(stakeProviderId))).unwrap();
+                        assert.equal(
+                            capacityStaked.remainingCapacity,
+                            currentStaked.remainingCapacity.toBigInt() + capacityMin,
+                             `should return a capacityLedger with ${capacityMin} remaining, got ${capacityStaked.remainingCapacity}`
+                        );
+                        assert.equal(
+                            capacityStaked.totalTokensStaked,
+                            currentStaked.totalTokensStaked.toBigInt() +  tokenMinStake,
```

Comment:
is this also 1n*CENTS?

Reply:
Yes, `tokenMinStake` is `1n*CENTS`. I see: it might be clearer if the staking on line 155 also used `tokenMinStake`, or this line didn't.